v63005/u_inref/gtm5059.csh

```diff
@@ -0,0 +1,111 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo "# Create a single region DB with region DEFAULT"
```

Comment:
Can you add a comment that a lot of the logic here is similar to v63004/u_inref/gtm1042.csh